### DISCUSSION TITLE
How to use different Tailwind configs in different templates?

### DISCUSSION THREAD

**Discussion by sinanyumer** [maizzle/framework#1253] [Q&A] [answered]
Maizzle Version: 4.8.4
Node.js Version: 20.11.0

I want to use different Tailwind configs in different templates but I don't know how to do that. Let's say I have two different Tailwind configs:

tailwind.A.config.js
tailwind.B.config.js

I want to use config A in template-1.html and config B in template-2.html

Is there a way to do that?

**Comment by sinanyumer** [comment, marked as answer]
I have found the solution. This works: https://maizzle.com/docs/configuration/tailwindcss#config

Like this:
```
template-1.html

---
build:
  tailwind:
    config: tailwind.A.config.js
---
```

**Reply by cossssmin** [reply]
Yup that’ll work 👍

In Maizzle 5 you’ll also be able to do it right from your `<style>` tag with `@config`:

https://tailwindcss.com/docs/configuration#using-multiple-configurations
